<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginRegistration</class>
 <widget class="QMainWindow" name="LoginRegistration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>575</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>575</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>575</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoginRegistration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
	background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
}

#RegistrationFrame{
	background: #fff;
	border-radius: 30px;
}

#RegistrationLabel , #LoginLabel{
	font: 25 italic 30pt &quot;Nork New&quot;;
}

#RegistrationInfoLabel{
	font: 15pt &quot;Russian Chance&quot;;
}

#InfoName_1, #InfoName_2, #InfoName_3, #InfoName_4, #InfoName_5{
	font: 75 italic 11pt &quot;Russian Bodoni&quot;;
}

#RegistrationPushButton{
  	font: 75 14pt &quot;Russian Antiqua&quot;;
 	background-color: white; 
  	color: black; 
 	border: 2px solid #4CAF50;
}

#RegistrationPushButton:hover{
 	background-color: #4CAF50;
  	color: white;
}

#RegistrationErrorList{
    background:#4CAF50;
	font: italic 11pt &quot;Aramian Normal&quot;;
    color:white;
}

#LoginErrorList{
	background: rgb(135, 0, 0);
	font: italic 11pt &quot;Aramian Normal&quot;;
	color:white;
}

#LoginFrame{
	background: #fff;
	border-radius: 30px;
   	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 		rgba(153, 63, 63, 255), stop:1 rgba(255, 255, 255, 255));
}

#LoginPushButton{
	 font: 75 14pt &quot;Russian Antiqua&quot;;
 	 background-color: white; 
 	 color: black; 
 	 border: 2px solid rgb(135, 0, 0);
}

#LoginPushButton:hover{
 	 background-color:  rgb(135, 0, 0);
 	 color: white;
}

#LoginEmailLineEdit , #LoginPasswordLineEdit {
	font: 14pt &quot;MV Boli&quot;;
}

#RegistrationLink , #LoginLink{
	background-color: #e7e7e7;
 	font: 14pt &quot;MV Boli&quot;;
 	color: white;
  	border:none;
}

#RegistrationLink:hover , #LoginLink:hover {
	background-color: white;
 	color: black;
	border: 2px solid #e7e7e7;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="RegistrationFrame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>401</width>
      <height>491</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="RegistrationPushButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>400</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Sing Up</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>401</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="RegistrationLabel">
        <property name="text">
         <string>Register</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="RegistrationInfoLabel">
        <property name="text">
         <string>Please fill in the form to create an account</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>115</y>
       <width>321</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="InfoName_1">
        <property name="text">
         <string>First Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="FirstNameLineEdit">
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="InfoName_2">
        <property name="text">
         <string>Last Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LastNameLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="InfoName_3">
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EmailLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="InfoName_4">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PasswordLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="InfoName_5">
        <property name="text">
         <string>Confirm Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ConfPasswordLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="RegistrationErrorList">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>400</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Please fill in all fields</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="LoginLink">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>442</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sign_in</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="LoginFrame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>401</width>
      <height>491</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="LoginLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>411</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>411</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="LoginEmailLineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LoginPasswordLineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoginPushButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="LoginErrorList">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Fill in all fields</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="RegistrationLink">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>300</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sign_Up</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
